vars/iqeUtils.groovy

```diff
@@ -464,28 +430,33 @@ private def createTestStages(String appName, Map appConfig) {
         configIQE(appOptions)
     }
 
-    stage("Install red-hat-internal-envs plugin") {
-        sh "iqe plugin install red-hat-internal-envs"
-    }
+    stage("Install dependencies") {
+        when {
+            expression { not appOptions['skipInstall'] }
+        }
+        stage("Install red-hat-internal-envs plugin") {
+            sh "iqe plugin install red-hat-internal-envs"
```

Comment:
I believe this stage can be removed. We include `red-hat-internal-envs` in `iqe-core` and update on every release.

Reply:
@quarckster im only leaving it in as thats how it currently is - once this works i can consider functionality changes beyond skipping all installs and